command.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package updater
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/keybase/go-logging"
+)
+
+func timerWithTimeout(timeout time.Duration, cmd *exec.Cmd, log logging.Logger) *time.Timer {
+	return time.AfterFunc(timeout, func() {
+		if cmd != nil && cmd.Process != nil {
+			log.Warningf("Command timed out, killing")
+			if err := cmd.Process.Kill(); err != nil {
+				log.Warningf("Error trying to kill process: %s", err)
+			}
+		}
+	})
```

Comment:
Yeah I removed the separate method, so it's inline now so it should be more readable.

So actually capturing output without using Output() or CombinedOutput requires a bunch of new code..e.g. calling Start(), Wait() then reading from the pipes.

I don't know the timer code seems legit and simple to me?